app/controllers/sessions_controller.rb

```diff
@@ -9,6 +9,7 @@ def create
       redirect_to root_url
     else
       user = User.find_by(email: params[:session][:email].downcase)
+      binding.pry
       if user && user.authenticate(params[:session][:password])
```

Comment:
The user class doesn't have a method ```authenticate```  I'm not sure where it went or what to put here.  If you know, please let me know, o.w. I'll take a shot this weekend.